students/jingwan/users/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from users.models     import User
+from users.validator  import *
```

Comment:
- import 는 가장 긴 단어에서 한 칸만 띄우시면 됩니다.
- import 할 때 * 사용은 자제해 주세요. 사용하는 모듈을 직접 import 해주세요.

```suggestion
from django.http  import JsonResponse
from django.views import View

from users.models    import User
from users.validator import (
    username_validate,
    email_validate,
    password_validate,
    phone_number_validate
)
```